pkg/config/config.go

```diff
@@ -414,6 +414,10 @@ type GitHubMonitor struct {
 	StripSuffix string `json:"strip-suffix,omitempty" yaml:"strip-suffix,omitempty"`
 	// Filter to apply when searching tags on a GitHub repository
 	TagFilter string `json:"tag-filter,omitempty" yaml:"tag-filter,omitempty"`
+	// Filter to apply when searching tags on a GitHub repository
+	TagFilterPrefix string `json:"tag-filter-prefix,omitempty" yaml:"tag-filter-prefix,omitempty"`
+	// Filter to search tags on the Github tag results
+	TagFilterContains string `json:"tag-filter-contains,omitempty" yaml:"tag-filter-contains,omitempty"`
```

Comment:
Do we need these? I would expect TagFilter is expressive enough to match prefixes and substrings.